### ISSUE TITLE
[BUG] - prompt reader node causes search dialog corruption

### ISSUE THREAD

**Issue by dirtyapenz** [receyuki/comfyui-prompt-reader-node#51]
### Description

When this node is enabled I encounter strange behaviour from the search dialog. From the search dialog, when clicking upon the top right dropdown control (which has things like LATENT, MODEL, CLIP, etc in it) i encounter anomalous behaviour. First the desktop locked up, graphical artifacts spans over the screen. Restarting the shell let me try again. This time clicking the control displays the normal items but also has appended:
STRING
INT
FLOAT
*

euler,euler_ancestral,heun,heunpp2,etc...
normal,karras,exponential,sgn_uniform,etc...

### Reproduction steps

1. Fresh install comfyui
2. Fresh manual install comfyui-prompt-reader-node
3. Open comfyui and double click any none-node space to bring up search
4. Click the top right drop down control.

**Comment by dirtyapenz** [comment]
Different behavior on different browsers, edge on ubuntu went full weird with graphic anomaly. Firefox jsut shows the following appended to the normal list...
STRING
INT
FLOAT
*
<a comma delimited list of all models that are installed>
<a comma delimited list of all vae that are installed>
<a comma delimited list of all samplers>
<a comma delimited list of all schedulers>
<a comma delimited list of all LoRA that are installed>

Note, previous example above failed to show the models,vae,and LoRA but instead has blank space.

**Comment by dirtyapenz** [comment]
My gut is telling me that this could be related to the "prompt generator" node which presents vae, model name, scheduler etc.

**Comment by receyuki** [comment]
Since you are the first user to raise this issue, it's highly likely this problem is caused by a conflict between my custom nodes and other custom nodes. Perhaps you could provide me with a list of all the custom nodes you have installed and post some screenshots. 

About your last question,
>Not a python guy, but perhaps related to this sorta thing making input/return types out of data? Shouldn't that be a string or something?
>
>SDParameterGenerator.ckpt_list = folder_paths.get_filename_list("checkpoints")

Yes, `folder_paths.get_filename_list("checkpoints")` is actually an array of strings. The built-in nodes are written in the same way, so this is irrelevant to the issue you mentioned.
